README.md

```diff
@@ -11,29 +11,41 @@ This plugin provides the functionality of [stress-ng](https://github.com/ColinIa
 - mq
 - hdd
 
-## To run directly with the Arcaflow engine:
+## Input structure
 
-In order to run the [arca-stressng plugin](stressng_plugin.py) follow these steps:
+The input structure (YAML or JSON) expects a couple of root-level parameters, like `timeout`,
```

Comment:
The point is that the plugin accepts either format.

Reply:
> The point is that the plugin accepts either format.

Ha: yes, because it accepts YAML. I don't object to the redundancy, since the technicality might not occur to a lot of readers, but it *is* amusing.